association_backend/association/api/types.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Optional
+
+import strawberry
+
+from association.domain import entities
+
+
+@strawberry.type
+class SubscriptionResponse:
+    user_id: int
+    creation_date: datetime
+    state: str
+    stripe_session_id: str
+    due_date: Optional[datetime]
+    stripe_id: Optional[str]
+    stripe_customer_id: Optional[str]
```

Comment:
this is the response of the mutation? Ideally, this should live in the mutation file

Reply:
otherwise, if this is the type of the Subscription model/entity, don't call it `SubscriptionResponse` but `Subscription`

Reply:
This was used by the Query too.
Now i moved it in the mutation and I renamed it in `Subscription`